src/components/List/index.js

```diff
@@ -4,33 +4,40 @@ import React, { Component } from 'react'
 import { ListGroup, Pagination } from 'react-bootstrap'
 import './styles.css'
 
+export type Props = {
+  pageSize: number,
+  total: number,
+  activePage: number,
+  onPageSelect?: Function,
+}
+
 class List extends Component {
-  constructor(props) {
+  constructor(props: Props) {
     super(props)
-    this.state = { activePage: this.props.activePage }
     this.handleSelect = this.handleSelect.bind(this)
```

Comment:
I think we should use use class property syntax instead of doing this in all of our constructors. And flow will be very happy, no need for the nasty `(this:any). handleSelect ` workaround anymore.

https://babeljs.io/docs/plugins/transform-class-properties/

So basically remove this `this.handleSelect = this.handleSelect.bind(this)` and use 
`handleSelect = (eventKey: number) =>`